src/Model/Order/Order.php

```diff
@@ -131,11 +140,28 @@ public static function create(
 
     private function applyOrderWasCreated(OrderWasCreated $event): void
     {
-        $this->id              = $event->getId();
-        $this->businessName    = $event->getBusinessName();
+        $this->id = $event->getId();
+        $this->businessName = $event->getBusinessName();
         $this->businessAddress = $event->getBusinessAddress();
-        $this->contactPerson   = $event->getContactPerson();
-        $this->isPaid          = $event->getHasPaid();
-        $this->amount          = $event->getAmount();
+        $this->contactPerson = $event->getContactPerson();
+        $this->isPaid = $event->getIsPaid();
+        $this->amount = $event->getAmount();
+    }
+
+    public function pay(NonNullDatePaid $datePaid): void
```

Comment:
In the domain modelling session we decided that you could not specify the date the order was paid. The date would be taken from when the event is persisted.

Reply:
Hint: we need to take the datetime from the event metadata.